agent/whois_ip_agent.py

```diff
@@ -41,21 +41,28 @@ def process(self, message: m.Message) -> None:
                 if not self.set_add('agent_whois_ip_asset', host):
                     logger.info('target %s was processed before, exiting', host)
                     return
-                mask = '/32'
+
+                mask = '32'
```

Comment:
This agent needs to be refactored for cleaner code. Use ipaddress to access the network without handling the case manually.